CalculatorAppJMS/CalculatorLibrary/CalculationsRecord.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CalculatorLibrary;
+
+public class CalculationsRecord
+{
+    public class OperationRecord //class that holds the list structure
+    {
+        public int Id { get; set; }

```

Comment:
Could you please apply consistent casing with your properties?

Reply:
Codacy sugested some changes to the properties here. I changed them. They should all be consistent now.